src/Storages/transformQueryForExternalDatabase.cpp

```diff
@@ -329,14 +329,26 @@ String transformQueryForExternalDatabaseImpl(
         }
         else if (auto * function = original_where->as<ASTFunction>())
         {
-            if (function->name == "and")
+            if (function->name == "and" || function->name == "tuple")
```

Comment:
I don't really get where `tuple` comes from in your example. Could you please add a testcase?

Reply:
`((a > 10) AND (toUInt64OrZero(timestamp_) > 12345)) ` is a `tuple`.